src/lib/utils/universal/signerTransforms.ts

```diff
@@ -127,21 +89,20 @@ class ViemSigner extends Signer {
       // https://viem.sh/docs/actions/wallet/prepareTransactionRequest.html#account-hoisting
       requestData
     )
-    const serializedTransaction = await this.walletClient.signTransaction(
-      request
-    )
-    const hash = await this.publicClient.sendRawTransaction({
-      serializedTransaction,
+    const hash = await this.walletClient.sendTransaction({
+      ...request,
```

Comment:
Is this to create a copy and make sure `request` is not changed?
It is viem's responsibility to ensure the function doesn't create side effects to `request`, and by the look of its code we should pass `request` directly
https://github.com/wevm/viem/blob/main/src/actions/wallet/sendTransaction.ts#L121

unless we are passing additional params than `request`, I don't see why we should spread it